### ISSUE TITLE
SCSS breakpoint error

### ISSUE THREAD

**Issue by vandelay87** [mac81/pure-drawer#46]
Pointing $show-drawer to $small-only results in the following error:

Error: You may not @extend an outer selector from within @media.
       You may only @extend selectors within the same directive.

Error in the _effect-bounce.scss file.

**Comment by mac81** [comment]
Yup. That's a mistake on my part. Replacing @extend with @include should fix it. I will update the project when I have time, although regretfully it may take a while.

Thanks for pointing out the issue.

**Comment by vandelay87** [comment]
No worries. I found out that if you only import the scss effect that you wish to use and not any of the others, it also resolves the error.
